backend/app/assets/stylesheets/spree/backend/sections/_orders.scss

```diff
@@ -1,18 +1,3 @@
-.line-item.editing {
-  .edit-line-item,
-  .delete-line-item,
-  .line-item-qty-show {
-    display: none !important;
-  }
-}
-
-.line-item:not(.editing) {
-  .save-line-item,
-  .cancel-line-item,
-  .line-item-qty-edit {
```

Comment:
`.save-line-item` and `.cancel-line-item` are still in use and also used in specs (a more dubious use).

I'm not sure they aren't appropriate classes to have.